consoles/video_stream.pm

```diff
@@ -218,7 +218,7 @@ sub _receive_frame_ffmpeg ($self) {
 
     # support P6 only
     if (!($header =~ m/^(P6\n(\d+) (\d+)\n(\d+)\n)/)) {
-        die "Invalid PPM header: $header";
+        die "Invalid PPM header: $header";    # uncoverable statement
```

Comment:
I think we are able to cover this by mocking the startup of ffmpeg and having it read some garbage data instead.

Reply:
Ok, let me try to mock ffmpeg startup

Reply:
I managed to make it die without mocking ffmpeg. I just supplied a png file instead of ppm file as another extra test

Reply:
Even better :-)